### PR TITLE
Major List 2022 Term 1

### LINKED ISSUE #1: Updates to Major List (2022-2023 Term 1)
## TLDR
Update Major List as of 2022-2023 Term 1

## Background
As William the "Toxic L God" stated, changes have been made to Lists of Majors

## Short-term solution
Update `src\static\major.json` according to [RES Program Codes](http://www.res.cuhk.edu.hk/zh-tw/general-information/program-codes)

## Long-term solution
Update automatically/fetch programme codes and lists according to any CUHK APIs or crawl from CUHK websites

### PR DESCRIPTION
**Pull request by ansonscleung**
Quick patch to update programme list as of 2022-2023 Term 1 according to [RES Programme Codes List](http://www.res.cuhk.edu.hk/en-gb/general-information/program-codes)

### CODE REVIEW

File: src/static/major.json [hunk 1]
```diff
@@ -420,12 +438,6 @@
       "chineseName": "性別研究（兩年制課程）",
       "faculties": ["SSF"]
     },
-    {
-      "code": "GLBSN",
-      "englishName": "Global Studies (2-year programme)",
-      "chineseName": "全球研究（兩年制課程）",
-      "faculties": ["SSF"]
-    },
```
Comment (LCM288) [inline src/static/major.json]: Should we keep old programs for backward compatibility?
And also add "inactive: true" to them?

File: src/static/major.json [hunk 2]
```diff
@@ -487,16 +499,10 @@
...
-      "code": "GLEFN",
-      "englishName": "Global Economics and Finance",
-      "chineseName": "環球經濟與金融",
-      "faculties": ["IDM", "BAF", "SSF"]
-    },
-    {
-      "code": "QFRMN",
-      "englishName": "Quantitative Finance and Risk Management Science",
-      "chineseName": "計量金融學及風險管理科學",
-      "faculties": ["IDM", "BAF", "SCF"]
```
Comment (LCM288) [inline src/static/major.json:499]: Same as above

**Review by LCM288 — CHANGES_REQUESTED** [review]

**Comment by LCM288** [comment]
cls 收咗工都要做code review :(

File: src/static/major.json (hunk 2 above)
Comment (ansonscleung) [reply src/static/major.json:499]: GLEFN and QFRMN were moved to a chronological order within IDM, so they are still on the list

File: src/static/major.json (hunk 1 above)
Comment (ansonscleung) [reply src/static/major.json]: Sure, and let me think think whether to use boolean or time
PS: AFAIK GLBSN merged into senior-entry GLSDN some years ago, maybe that's why the code is deleted from RES?

Comment (LCM288) [reply src/static/major.json]: Or let me check if this code is used right now

Comment (LCM288) [reply src/static/major.json]: I don't find any one with the program, let's remove it directly.

**Review by LCM288 — APPROVED** [review]

**Pull request merged**